portfolio.html

```diff
@@ -21,12 +21,60 @@ <h1>Welcome to my portfolio</h1>
 			<p>Hello, this is a portfolio of some of my work that I have done</p>
 		</div>
 	</section>
-	<section class="about" id="about">
+	<a class="anchor" id="about"></a>
+	<section class="about">
 		<div class="abouttxt">
 			<h2 class="abouth">About Me</h2>
 			<p class="aboutp">Hi, I’m Marcus. I am a student at iO academy on the full stack track course, I am currently working on this website as part of my first project week. By the end of this course I will be able to make something much better than this so it does not really matter what I write here because even if I do use this site as my actual portfolio I will change everything else I just need content to fill a box which is why I have not used any abbreviations like “don’t” or “haven’t” and I have actually written out “do not” and “have not” to make it longer.</p>
 		</div>
 		<div class="aboutimg"></div>
 	</section>
+	<a class="anchor" id="portfolio"></a>
+	<section class="portfolio">
+		<div class="Portfolioh">
+			<h2>Portfolio</h2>
+			<p>This is a portfolio of some of my work</p>
+		</div>
+		<div class="projecta">
+			<div class="projecttxt">
+				<h3>Project 1</h3>
+				<p>This is the text for my first project, it is very similar to the text for the other projects as I will just copy and paste it and change it so that they say “first”, “second”, “third” and “forth” in each of the project boxes.</p>
+			</div>
+			<div class="projectimg"></div>
+		</div>
+		<div class="projectb">
```

Comment:
Is there a reason you decided to give each project different classes (projecta,b,c,d) instead of the same class? Are you applying slightly different styles to each project?